electron-app/main/menu.ts

```diff
@@ -1,88 +1,92 @@
+//     label: 'Edit',
+//     submenu: new [
+//       { role: 'undo' },
+//       { role: 'redo' },
+//       { type: 'separator' },
+//       { role: 'cut' },
+//       { role: 'copy' },
+//       { role: 'paste' },
+//       { role: 'pasteandmatchstyle' },
+//       { role: 'delete' },
+//       { role: 'selectall' }
+//     ]()
+//   },
+//   {
+//     label: 'View',
+//     submenu: [
+//       { role: 'reload' },
+//       { role: 'forcereload' },
+//       { type: 'separator' },
+//       { role: 'resetzoom' },
+//       { role: 'zoomin' },
+//       { role: 'zoomout' },
+//       { role: 'togglefullscreen' },
+//       { type: 'separator' },
+//       { role: 'toggledevtools' }
+//     ]
+//   }
+// ];
 
-const HELP_MENU: MenuItemConstructorOptions = {
-  role: 'help',
-  submenu: [
-    {
-      label: `v${packageJson.version}`,
-      enabled: false
-    },
-    { type: 'separator' },
-    {
-      label: 'Help / FAQ',
-      click() {
-        shell.openExternal('https://support.mycrypto.com/');
-      }
-    },
-    {
-      label: 'Report a Bug',
-      click() {
-        shell.openExternal(`${REPOSITORY}/issues/new`);
-      }
-    }
-  ]
-};
+// const HELP_MENU: MenuItemConstructorOptions = {
+//   role: 'help',
+//   submenu: [
+//     {
+//       label: `v${packageJson.version}`,
+//       enabled: false
+//     },
+//     { type: 'separator' },
+//     {
+//       label: 'Help / FAQ',
+//       click() {
+//         shell.openExternal('https://support.mycrypto.com/');
+//       }
+//     },
+//     {
+//       label: 'Report a Bug',
+//       click() {
+//         shell.openExternal(`${REPOSITORY}/issues/new`);
+//       }
+//     }
+//   ]
+// };
 
-if (process.platform === 'darwin') {
-  MENU.unshift({
-    label: APP_TITLE,
-    submenu: [
-      { role: 'about' },
-      { type: 'separator' },
-      { role: 'hide' },
-      { role: 'hideothers' },
-      { role: 'unhide' },
-      { type: 'separator' },
-      { role: 'quit' }
-    ]
-  });
+// if (process.platform === 'darwin') {
+//   MENU.unshift({
+//     label: APP_TITLE,
+//     submenu: [
+//       { role: 'about' },
+//       { type: 'separator' },
+//       { role: 'hide' },
+//       { role: 'hideothers' },
+//       { role: 'unhide' },
+//       { type: 'separator' },
+//       { role: 'quit' }
+//     ]
+//   });
 
-  // Modified help menu
-  MENU.push({
-    ...HELP_MENU,
-    submenu: [
-      ...(HELP_MENU.submenu as MenuItemConstructorOptions[]),
-      {
-        label: 'Speech',
-        submenu: [{ role: 'startspeaking' }, { role: 'stopspeaking' }]
-      }
-    ]
-  });
-} else {
-  MENU.push(HELP_MENU);
-}
+//   // Modified help menu
+//   MENU.push({
+//     ...HELP_MENU,
+//     submenu: [
+//       ...(HELP_MENU.submenu as MenuItemConstructorOptions[]),
+//       {
+//         label: 'Speech',
+//         submenu: [{ role: 'startspeaking' }, { role: 'stopspeaking' }]
+//       }
+//     ]
+//   });
+// } else {
+//   MENU.push(HELP_MENU);
+// }
```

Comment:
Is there a reason to keep all of this commented stuff?

Reply:
I was wondering what it was used for.
But it definitely looks like dead code.
If It is no commented, it messes up.

Would you suggest to remove this code without any thought ?

Reply:
Can you check what it was for? 
I suspect it was the side menu of the previous designs